src/Symfony/Component/Translation/TranslatorInterface.php

```diff
@@ -20,6 +20,8 @@
  */
 interface TranslatorInterface
 {
+    public const DEFAULT_DOMAIN = 'messages';
```

Comment:
there are more occurences where this value is hardcoded and should be replaced by the constant